src/coreclr/utilcode/guidfromname.cpp

```diff
@@ -187,10 +187,10 @@ void GuidFromName
 }
 
 
-// This guid is used for calling GuidFromName function as COM+ runtime uniqualifier
+// This guid is used for calling GuidFromName function as CLR runtime uniqualifier
```

Comment:
```suggestion
// This guid is used for calling GuidFromName function as CLR uniqualifier
```
R in CLR stands for runtime, so "CLR runtime" would be "Common Language Runtime Runtime"